docs/learning/tutorials/cra-template-imodeljs.md

```diff
@@ -0,0 +1,27 @@
+## Developing a create-react-app with iModeljs template
+
+### Setup
+
+- [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
+- Have access to a cloud hosted iModel. If you do not have access to one, follow one of our tutorials to [create an online iModel]($docs/learning/tutorials/index.md)
+- To use this template, add `--template @bentley/cra-template-imodeljs` when creating a new app. You should also use the @bentley/react-scripts scripts version to compile your application.
```

Comment:
Can we update the wording here to make it similar to the explicit instructions in #307 where it walks through how to use react-scripts partially as well? Hopefully that will make it clearer how to use it for someone new to CRA templates.

Reply:
Modified it, but I need to give links for the npm package and the git repo (for cra-template-iModeljs) when [that pull request](https://github.com/imodeljs/create-react-app/pull/25) is completed.

Reply:
Let me know if any more changes are needed.

Reply:
@calebmshafer we will address reorganizing the tutorials page in a subsequent PR.